src/panels/config/cloud/account/cloud-tts-pref.ts

```diff
@@ -16,12 +16,30 @@ import type { CloudTTSInfo } from "../../../../data/cloud/tts";
 import {
   getCloudTTSInfo,
   getCloudTtsLanguages,
-  getCloudTtsSupportedVoices,
 } from "../../../../data/cloud/tts";
 import { showAlertDialog } from "../../../../dialogs/generic/show-dialog-box";
 import type { HomeAssistant } from "../../../../types";
 import { showTryTtsDialog } from "./show-dialog-cloud-tts-try";
 
+export const getCloudTtsSupportedVoices = (
+  language: string,
+  info: CloudTTSInfo | undefined
+) => {
+  const voices: { voiceId: string; voiceName: string }[] = [];
+
+  if (!info) {
+    return voices;
+  }
+
+  for (const [curLang, voiceId, voiceName] of info.languages) {
+    if (curLang === language) {
+      voices.push({ voiceId, voiceName });
+    }
+  }
+
+  return voices;
```

Comment:
Do we no longer want to sort them?

Reply:
It's also better to keep this code on the original data file for reusability.

Reply:
The data now comes sorted from the server.

https://github.com/NabuCasa/hass-nabucasa/blob/master/scripts/update_voice_data.py#L56-L64